docs/components/connectors/out-of-the-box-connectors/outbound/aws-lambda.md

```diff
@@ -4,7 +4,7 @@ title: AWS Lambda Connector
 description: Invoke AWS Lambda function.
```

Comment:
```suggestion
description: Invoke AWS Lambda functions with an outbound Connector.
```

Little bit of scope creep, but while I'm here, I thought this needed a little something else.